### PR TITLE
PLAT-1210 Allow custom DockerFile 

### PR DESCRIPTION
**Pull request by andyloughran**
Allows a custom dockerfile to be submitted - defaults to the Dockerfile

### CODE REVIEW

File: scripts/build-tag-push-ecr.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ echo "building image(s)"
 cd "${WORKING_DIRECTORY}"
 echo "Packaging app in /$WORKING_DIRECTORY"
 
-docker build -t "$ECR_REGISTRY/$ECR_REPO_NAME:$GITHUB_SHA" .
+docker build -t "$ECR_REGISTRY/$ECR_REPO_NAME:$GITHUB_SHA" -f $DOCKERFILE .
```
Comment (CharlesIC) [inline scripts/build-tag-push-ecr.sh:10]:
```suggestion
docker build -t "$ECR_REGISTRY/$ECR_REPO_NAME:$GITHUB_SHA" -f "$DOCKERFILE" .
```

Should quote this to be safe

Comment (andyloughran) [reply scripts/build-tag-push-ecr.sh:10]: Cheers @CharlesIC

File: action.yaml [hunk 2]
```diff
@@ -1,10 +1,9 @@
 name: "Upload Containers to ECR, update references in CF templates"
 description: "Packages and uploads, replaces reference in (eg) template.yaml"
 inputs:
-
   role-to-assume-arn:
     description: "The secret with the ARN of the role to assume (required) eg secrets.GH_ACTIONS_ROLE_ARN"
-    required: true
+    required: true"
```
Comment (CharlesIC) [inline action.yaml:6]:
```suggestion
    required: true
```
Oops

File: scripts/build-tag-push-ecr.sh [hunk 3]
```diff
@@ -4,16 +4,15 @@ set -eu
 
 echo "building image(s)"
 
-cd "${WORKING_DIRECTORY}"
-echo "Packaging app in /$WORKING_DIRECTORY"
+echo "Packaging sam app in /$WORKING_DIRECTORY"
```
Comment (CharlesIC) [inline scripts/build-tag-push-ecr.sh:7]:
```suggestion
echo "Packaging sam app in $WORKING_DIRECTORY"
```
This throw me off before

**Review by devdutoitGDS — APPROVED** [review]

**Pull request merged**